zenduty/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Could you regenerate the manifest file? It looks like it's missing the required `app_uuid` that should be present in all manifests. Ex: https://github.com/DataDog/integrations-extras/blob/master/amixr/manifest.json#L3

Reply:
Wasn't aware there was a generator for the manifest file, can you link a relevant resource that helps me generate a uuid?

Reply:
Fixed that @yzhan289, thanks!